BICCN-policies/DRAFT Citation and identifier policy.md

```diff
@@ -0,0 +1,49 @@
+# Policy regarding support of data and resource citation and use of persistent identifiers
+
+## For data submitters/authors
+
+### Persistent identifiers + metadata for BICCN datasets
```

Comment:
We should define "dataset" in this document somewhere. A dataset is a "defined set of data" that should be referenced together. For the purpose of the BICCN inventory, it is a set of data generated in the same way and by the same lab.  For the purpose of a paper, it is a slice of that data.